frontend/src/components/SynonymRecognition.vue

```diff
@@ -0,0 +1,76 @@
+<template>

```

Comment:
For future reference, with Layout Frameworks like Bootstrap, Bootstrap Vue, Vuetify, Material Design, Material UI etc, 
Tables can simply be implemented with no hard coding html. simply having some objects in your data field and a link to it in an html tag does the trick

eg: 
![image](https://user-images.githubusercontent.com/45756873/112076338-81ad1c00-8b37-11eb-99ca-0b24aab60338.png)

https://bootstrap-vue.org/docs/components/table

Probably would have took a quarter of the time to implement with a better result.